src/_test_/calculator.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Calculator from '../components/Calculator';
```

Comment:
- [ ] Good job trying to fix some changes required previously. However, As mentioned by the previous reviewer and for the project requirements, kindly ensure that you use `react testing-library` for your tests as no test is used. **React Testing Library should be used to simulate user interaction.** I suggest you refer **[this link to have idea of how to proceed](https://testing-library.com/docs/react-testing-library/cheatsheet/)**.